<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>160</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="kd_obatLabel">
       <property name="text">
        <string>kd_obat</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="kd_obatLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nama_obatLabel">
       <property name="text">
        <string>nama_obat</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nama_obatLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="hargaLabel">
       <property name="text">
        <string>harga</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="hargaLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stokLabel">
       <property name="text">
        <string>stok</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="stokLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="keteranganLabel">
       <property name="text">
        <string>keterangan</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="keteranganLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Simpan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ubah</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>kd_obat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nama_obat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>harga</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>stok</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>keterangan</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
